packages/cli/src/actions/deployPool.ts

```diff
@@ -15,14 +16,13 @@ const deployPool = async (opts: Options): Promise<void> => {
   console.log("Sending transaction from", signer.address);
 
   const market = await sdk.models.fetchMarketData(marketId);
-  const res = await market.deploySwapPool(signer, [
-    "25".concat("0".repeat(10)),
-    "125".concat("0".repeat(9)),
-    "125".concat("0".repeat(9)),
-  ]);
+
+  const weights = Array(market.outcomeAssets.length+1)
+    .fill("1".concat("0".repeat(10)));
```

Comment:
This is probably not the default that we want, usually we want to set the initial weights to 50% ZTG and 50% all other assets evenly, that way we set all other assets to equal prices at the beginning of the market.

Reply:
Good to know.
This is just a hack, however for faster testing.
Once the SDK functions are tested working reliably, I will add both a weights param and a sensible default into the CLI